add.py

```
@@ -0,0 +1,5 @@
+def add(first_number:int,secondName:int)->str:
+    return first_number + second_number
```

Ruff (0.8.2)

Undefined name `second_number`

(F821)